vibi-dpu/src/core/review.rs

```diff
@@ -34,16 +34,36 @@ pub async fn process_review(message_data: &Vec<u8>) {
 		return;
 	}
 	log::info!("[process_review] Processing PR : {}", &review.id());
-	let access_token_opt = get_access_token(&review).await;
-	if access_token_opt.is_none(){
-		log::error!("[process_review] empty access_token_opt");
+	let pat_env_var = "GITHUB_PAT";
+    let provider_env_var = "PROVIDER";
+	let mut access_token: Option<String> = None;
+    if let (Ok(pat), Ok(provider)) = (env::var(pat_env_var), env::var(provider_env_var)) {
```

Comment:
Use our coding conventions here, like declaring a res variable, checking if its none etc and logging error/ returning